dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -456,21 +497,64 @@ export const DynamicPackage = ({
 			secondSlice = cards.slice(0, 1);
 			thirdSlice = cards.slice(1);
 			break;
+		case 2:
+			layout = 'twoStandards';
+			firstSlice = snaps;
+			secondSlice = cards.slice(0, 2);
+			break;
+		case 1:
+			layout = 'oneStandard';
+			firstSlice = snaps;
+			secondSlice = cards.slice(0, 1);
+			break;
 		default:
 			if (cards[0].isBoosted) {
-				layout = 'primaryBoosted';
+				layout = 'threeOrFourStandardsBoosted';
 				firstSlice = snaps;
 				secondSlice = cards.slice(0, 1);
 				thirdSlice = cards.slice(1);
 			} else {
-				layout = 'fourOrLessStandards';
+				layout = 'threeOrFourStandards';
 				firstSlice = snaps;
 				secondSlice = cards;
 			}
 	}
 
 	switch (layout) {
-		case 'primaryBoosted':
+		case 'oneStandard':
+			return (
+				<>
+					<Snap100
+						snaps={firstSlice}
+						containerPalette={containerPalette}
+						showAge={showAge}
+					/>
+					<Card100
```

Comment:
I was actually waiting for this PR to be merged so that I could do this, but I think you've already done it, thanks!

This is fixes https://github.com/guardian/frontend/issues/25194 on DCR right?

Reply:
Yes it does :)